.github/script/get-prev-version

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env node
```

Comment:
where do we use this script?

Reply:
Roster test doesn't need this script. Deleted the file.